src/pybind/mgr/dashboard_v2/frontend/src/app/core/not-found/not-found.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'cd-not-found',
+  templateUrl: './not-found.component.html',
+  styleUrls: ['./not-found.component.scss']
+})
+export class NotFoundComponent implements OnInit {
```

Comment:
If ngOnInit() is not used, then we do not have to implement that interface OnInit.

Reply:
done.